packages/client/src/__tests__/integration/errors/referentialActions-onDelete-default-foreign-key-error-sqlserver/test.ts

```diff
@@ -50,15 +57,15 @@ describe.skip('referentialActions-onDelete-default-foreign-key-error(sqlserver)'
     } catch (e) {
       expect(e.message).toMatchInlineSnapshot(`
 
-Invalid \`prisma.user.delete()\` invocation in
-/client/src/__tests__/integration/errors/default-onDelete-cascade-sqlserver/test.ts:41:31
+        Invalid \`prisma.user.delete()\` invocation in
+        /client/src/__tests__/integration/errors/referentialActions-onDelete-default-foreign-key-error-sqlserver/test.ts:0:0
 
-  38 expect(await prisma.user.findMany()).toHaveLength(1)
-  39 
-  40 try {
-→ 41   await prisma.user.delete(
-  The change you are trying to make would violate the required relation 'PostToUser' between the \`Post\` and \`User\` models.
-`)
+           49 expect(await prisma.user.findMany()).toHaveLength(1)
+           50 
+           51 try {
+        →  52   await prisma.user.delete(
+          Foreign key constraint failed on the field: \`Post_authorId_fkey (index)\`
```

Comment:
Interesting that the error message change here, right? Expected?

Reply:
Not really, it's the same in other tests (so expected) and it was outdated